### PR TITLE
use lts-19.27 (up from 16.21)

### PR DESCRIPTION
**Pull request by petertseng**
We would do well to keep this in sync with the Haskell track's resolver. Otherwise, a student might use something that's in 19.27 and not in 16.21, and then their tests could succeed locally but fail on the test runner! We wouldn't want that to happen.

https://github.com/exercism/haskell/pull/1117

https://www.stackage.org/diff/lts-16.21/lts-19.27

### CODE REVIEW

**Comment by petertseng** [comment]
(this comment is not related to this PR) interesting; GitHub started removing the line wraps from pull request descriptions! See how following normal rules for commit messages commit messages are wrapped to a certain number of characters per line, but the corresponding line in the description is not wrapped at all.

**Comment by petertseng** [comment]
I see that pull requests to this repository don't require approval before merging. I have no comment on whether that should continue to be the case but wanted to bring it up. For a change like this I would be satisfied with just the tests passing and not necessarily wait for approval, but in other cases I would not trust my own judgment and always wait for a second pair of eyes. It's a case-by-case thing

**Comment by ErikSchierboom** [comment]
> I see that pull requests to this repository don't require approval before merging. I have no comment on whether that should continue to be the case but wanted to bring it up. For a change like this I would be satisfied with just the tests passing and not necessarily wait for approval, but in other cases I would not trust my own judgment and always wait for a second pair of eyes. It's a case-by-case thing

I have a script ready to run that will sync protection settings for all track repos (including this one). We've just not gotten around to running it :(

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**
